test/integration/json.test.js

```diff
@@ -5,89 +5,117 @@ const chai = require('chai');
+    });
 
-        const user = await this.UserFields.create({
-          emergencyContact: { name: 'joe', phones: [1337, 42] },
-        });
+    it('should update json using a custom field name', async function () {
+      this.UserFields = this.sequelize.define('UserFields', {
+        emergencyContact: { type: DataTypes.JSON, field: 'emergy_contact' },
+      });
+      await this.UserFields.sync({ force: true });
 
-        expect(user.emergencyContact.name).to.equal('joe');
+      const user0 = await this.UserFields.create({
+        emergencyContact: { name: 'joe', phones: [1337, 42] },
       });
 
-      it('should update json using a custom field name', async function () {
-        this.UserFields = this.sequelize.define('UserFields', {
-          emergencyContact: { type: DataTypes.JSON, field: 'emergy_contact' },
-        });
-        await this.UserFields.sync({ force: true });
+      user0.emergencyContact = { name: 'larry' };
+      const user = await user0.save();
+      expect(user.emergencyContact.name).to.equal('larry');
+    });
 
-        const user0 = await this.UserFields.create({
-          emergencyContact: { name: 'joe', phones: [1337, 42] },
-        });
+    it('should be able retrieve json value as object', async function () {
+      const emergencyContact = { name: 'kate', phone: 1337 };
 
-        user0.emergencyContact = { name: 'larry' };
-        const user = await user0.save();
-        expect(user.emergencyContact.name).to.equal('larry');
-      });
+      const user0 = await this.User.create({ username: 'swen', emergency_contact: emergencyContact });
+      expect(user0.emergency_contact).to.deep.eq(emergencyContact);
 
-      it('should be able retrieve json value as object', async function () {
-        const emergencyContact = { name: 'kate', phone: 1337 };
+      const user1 = await this.User.findOne({ where: { username: 'swen' }, attributes: ['emergency_contact'] });
+      expect(user1.emergency_contact).to.deep.eq(emergencyContact);
+    });
 
-        const user0 = await this.User.create({ username: 'swen', emergency_contact: emergencyContact });
-        expect(user0.emergency_contact).to.eql(emergencyContact);
-        const user = await this.User.findOne({ where: { username: 'swen' }, attributes: ['emergency_contact'] });
-        expect(user.emergency_contact).to.eql(emergencyContact);
+    // TODO: enable on all dialects
+    // JSONB Supports this, but not JSON in postgres/mysql
+    if (current.dialect.name === 'sqlite') {
+      it('should be able to find with just string', async function () {
+        await this.User.create({
+          username: 'swen123',
+          emergency_contact: 'Unknown',
+        });
+
+        const user = await this.User.findOne({
+          where: {
+            emergency_contact: 'Unknown',
+          },
+        });
+
+        expect(user.username).to.equal('swen123');
       });
+    }
 
+    if (dialect.supports.jsonOperations) {
```

Comment:
jsonOperations was added by the DataTypes PR, but at the same time not including this is going to cause a huge diff issue in the other PR 😬

Reply:
I seem to have missed that one. I was planning on making a PR for jsonOperations support either way to reduce the DataTypes one a little bit so this will be fixed there